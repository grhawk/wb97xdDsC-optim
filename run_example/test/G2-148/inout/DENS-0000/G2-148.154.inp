 $BASIS GBASIS=N31 NGAUSS=6 $END
 $DATA
 Should be setted!
 C1
 C     6.00     0.000000     0.000000     0.000000
 $END
 $DFT NLEB=110 DDSC=.t. NRAD=24 $END
 $SYSTEM MWORDS=8 $END
 $SCF DIRSCF=.t. $END
 $CONTRL RUNTYP=ENERGY MAXIT=200 DFTTYP=wB97X ICHARG=0 MULT=3 
EXETYP=RUN SCFTYP=UHF $END