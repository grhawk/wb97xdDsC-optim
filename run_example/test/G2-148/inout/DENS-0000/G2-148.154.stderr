/var/spool/slurm/job10165/slurm_script: line 9: module: command not found
